src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -201,12 +201,7 @@ class Gold_Data_Spec() : EntitySpec<Gold_Data> {
         val schema = Schema(
             listOf(),
             SchemaFields(
-                singletons = mapOf(
-                    "num" to FieldType.Number,
-                    "txt" to FieldType.Text,
-                    "lnk" to FieldType.Text,
-                    "flg" to FieldType.Boolean
-                ),
+                singletons = mapOf("num" to FieldType.Number, "txt" to FieldType.Text, "lnk" to FieldType.Text, "flg" to FieldType.Boolean),
```

Comment:
This line looks too long for the lint check.

Reply:
Totally. 
I've updated the API to make it easier to handle this case.